receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -388,23 +389,61 @@ func addNumberDatapoints(datapoints pmetric.NumberDataPointSlice, ls labels.Labe
 		dp.SetDoubleValue(sample.Value)
 
 		attributes := dp.Attributes()
-		for _, l := range ls {
-			if l.Name == "instance" || l.Name == "job" || // Become resource attributes
-				l.Name == labels.MetricName || // Becomes metric name
-				l.Name == "otel_scope_name" || l.Name == "otel_scope_version" { // Becomes scope name and version
-				continue
-			}
-			attributes.PutStr(l.Name, l.Value)
-		}
+		extractAttributes(ls).CopyTo(attributes)
 	}
 }
 
 func addSummaryDatapoints(_ pmetric.SummaryDataPointSlice, _ labels.Labels, _ writev2.TimeSeries) {
 	// TODO: Implement this function
 }
 
-func addHistogramDatapoints(_ pmetric.HistogramDataPointSlice, _ labels.Labels, _ writev2.TimeSeries) {
-	// TODO: Implement this function
+func addExponentialHistogramDatapoints(datapoints pmetric.ExponentialHistogramDataPointSlice, ls labels.Labels, ts writev2.TimeSeries) {
+	for _, histogram := range ts.Histograms {
+		dp := datapoints.AppendEmpty()
+		dp.SetStartTimestamp(pcommon.Timestamp(ts.CreatedTimestamp * int64(time.Millisecond)))
+		dp.SetTimestamp(pcommon.Timestamp(histogram.Timestamp * int64(time.Millisecond)))
+		dp.SetSum(histogram.Sum)
+		// TODO: Mising steps to finish histogram conversion
```

Comment:
What's the plan for adding the remaining parts of the exponential histogram?

Reply:
After merge this one, will be my priority to close those TODOs that im leaving

Here, I just set the values that are simpler to deal with and validate if we can convert native histograms -> exponential histograms

Reply:
After talk with Sens, we decided to implement the whole Histograms on this PRs. I'm already take care of all missing TODOs